src/components/lib/VChip/VChip.vue

```diff
@@ -45,7 +45,7 @@
 </style>
 
 <script>
-import CancelIcon from "../../../assets/iconography/cancel.svg";
+import CancelIcon from "../../../assets/iconography/close.svg";
```

Comment:
Let's stick to root-relative imports everywhere.

```suggestion
import CancelIcon from "@/assets/iconography/close.svg";
```